### PR TITLE
Add support for bigint

### PR DESCRIPTION
**Pull request by mdoi2**
I have implemented support for bigint type 😄
The schema is output as integer(int64).

If I am missing any considerations, please point them out🙇🏻‍♂️

### CODE REVIEW

**Comment by samchungy** [comment]
Could you update the readme otherwise lgtm

**Comment by mdoi2** [comment]
Thanks! I had overlooked that.
And I realized that each validation process should be supported like ZodNumber.
I will try to implement it, so please wait.

**Comment by mdoi2** [comment]
Oh... I was wondering why only BIgInt is not supported, maybe because metadevpro/openapi3-ts does not support bigint and you dare not support it?

**Comment by samchungy** [comment]
> Thanks! I had overlooked that. And I realized that each validation process should be supported like ZodNumber. I will try to implement it, so please wait.

I'm not entirely sure what you mean by that to be honest. The contents of your PR seem fine to me

**Comment by mdoi2** [comment]
Thanks for the reply, samchungy.

maximum/minimum/multipleOf in SchemaObject is number type, so bigint cannot be set.
I have come up with the following 2 patterns to deal with this issue without changing the type.

- Exclude maximum/minimum/multipleOf from support and explain it in the README.
- Supports maximum/minimum/multipleOf only in the range of number; if the number exceeds the range, it will be excluded from the output and display warning using console.warn.

Because of the wide range of number, I think the latter will suffice for most of my use cases.
Please give me your opinion on this.

**Comment by mdoi2** [comment]
Thanks for your advice🙇🏻‍♂️ certainly agree with your thoughts.
If you want to use min/max, you can take the strategy of using `number` instead of `bigint`.
Fortunately, the range of `number` is wider than that of int32.

I have appended it to README.md.
Could you please check if everything is OK?

File: README.md [hunk 1]
```diff
@@ -763,6 +763,8 @@ As an example `z.string().nullable()` will be rendered differently
 - ZodAny
 - ZodArray
   - `minItems`/`maxItems` mapping for `.length()`, `.min()`, `.max()`
+- ZodBigInt
+  - `integer` `type` and `int64` `format` mapping for `.bigint()`
```
Comment (samchungy) [inline README.md:767]:
```suggestion
  - `integer` `type` and `int64` `format` mapping`
```

**Review by samchungy — APPROVED** [review]
Thanks!

**Pull request merged**